ci/vmtest/run_selftests.sh

```diff
@@ -54,21 +48,34 @@ read_test_names() {
     foldable end read_test_names
 }
 
-test_progs() {
-  foldable start test_progs "Testing test_progs"
+test_progs_helper() {
+  local suffix="$1"
```

Comment:
minor: if we address below comment, do we need suffix variable or can move it there?

Reply:
Agreed.